doc/source/getting_started/index.rst

```diff
@@ -23,7 +21,7 @@ Before you can use pandas, you’ll need to get it installed.
                 <div class="card-body">
                     <p class="card-text">
 
-Pandas is part of the `Anaconda <http://docs.continuum.io/anaconda/>`__ distribution and can be
+pandas is part of the `Anaconda <http://docs.continuum.io/anaconda/>`__ distribution and can be
```

Comment:
in #32388 , `pandas` is being written
```
*pandas*
```
 - should that be done here as well for consistency?

Reply:
We want to move `*pandas*` to `pandas` https://github.com/pandas-dev/pandas/issues/32316